fallback_builtins.h

```diff
@@ -41,4 +41,24 @@ static __forceinline unsigned long long __builtin_ctzll(uint64_t value) {
 
 #endif
 #endif
+
+#if defined(_M_ARM64) || defined(_M_ARM)
```

Comment:
This is wrong... This applies to Visual C++ only, but older gcc versions didn't support these either... I know at least gcc 9.4.0 and newer do...

Reply:
I kind of figured as much with the _M prefix.  Maybe I can use a define we setup with the configure/cmake harness?

Reply:
Hmm, I'll just use the SLIDEHASH one since that's where it's used.  Conceivably the correct thing to do might be to find a common preprocessor define for all compilers or maybe have a bunch of ||'s with other places it ends up being used.